### PR TITLE
Add ArchLinux ansible-test container

### PR DESCRIPTION
**Pull request by felixfontein**
A first quick test on adding another container. Probably needs some more cleanup.

### CODE REVIEW

**Comment by dmsimard** [comment]
@felixfontein I tested locally on fedora35 and it works for me:
```bash
> rpm -qa |egrep "buildah|podman"
podman-gvproxy-3.4.4-1.fc35.x86_64
podman-plugins-3.4.4-1.fc35.x86_64
podman-3.4.4-1.fc35.x86_64
buildah-1.23.1-1.fc35.x86_64
```

File: ansible-test/archlinux/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+coverage
```
Comment (dmsimard) [inline ansible-test/archlinux/requirements.txt:1]: Need to be pinned to 4.5.4 for now: https://github.com/ansible-community/images/issues/2

```suggestion
coverage==4.5.4
```

**Comment by dmsimard** [comment]
LGTM. I have built and uploaded it to https://quay.io/ansible-community/test-image:archlinux so that at least it exists -- it'll eventually be updated automatically.

**Comment by felixfontein** [comment]
ready_for_review

File: .github/workflows/ansible-test.yml [hunk 2]
```diff
@@ -38,9 +40,15 @@ jobs:
           ${{ matrix.name }}/build.sh
         working-directory: ./ansible-test/
 
+      # Right now it is not possible to use some of the images (Debian Bullseye, ArchLinux) with podman in GitHub Actions,
+      # while others (CentOS 8 Stream) work fine. This is not specific to the images in this repository, the official
+      # ansible-test Ubuntu 20.04 image also does not work with podman in GHA, but does work with Docker in GHA.
```
Comment (dmsimard) [inline .github/workflows/ansible-test.yml:45]: I'll create an issue so we don't forget about this being a known issue.

**Comment by dmsimard** [comment]
Works for me, the GHA is unfortunate but we can work around it for now. Let's figure it out later.

**Comment by dmsimard** [comment]
Thanks @felixfontein !

**Comment by felixfontein** [comment]
@dmsimard thanks for reviewing and merging!

**Pull request merged**